docs/tails_printing_guide.rst

```diff
@@ -7,12 +7,75 @@ potentially untrusted network. This is why **we strongly recommend using
+
+We `maintain a list of printers
+<https://docs.securedrop.org/en/stable/hardware.html#printers>`_ that we have
+personally tested and gotten to work with Tails, in the Hardware guide; if
+possible, we recommend using one of those printers. The Linux Foundation also
+maintains the `OpenPrinting database <https://www.openprinting.org/printers>`_,
+which documents the compatibility, or lack thereof, of numerous printers from
+most every manufacturer.
+
+.. note:: The latest generations of printers might or might not be represented
+          by the OpenPrinting database; also, the database does not document
+          whether or not a printer is wireless, so this will involve manually
+          checking models of interest, if you wish to use this resource as a
+          guide for purchasing a non-wireless printer suitable for use with
+          SecureDrop.
+
+With that in mind, this database is arguably the best resource for researching
+the compatibility of printers with Linux. As a tip for narrowing down your
+search, look for printers that are compatible with Debian, or Debian-based
+distributions like Ubuntu, since Tails itself is also Debian-based. This might
+increase the chances for a successfully seamless installation experience in
+Tails.
+
+In any case, this document outlines the usual set of steps that we follow when
+attempting to use a new printer with Tails, and provides some troubleshooting
+tips that you may find useful if you are trying to use a different printer.
+
+Installing and Printing via the Tails GUI
+-----------------------------------------
+
+Let's look at Tails 3.0's typical flow for installing a USB-connected printer.
+
+First, connect the printer to your Tails-booted computer via USB, then turn the
+printer on.
+
+Now you'll want to open your "Settings" panel, and double-click "Printers".
+
+|select printer from settings|
+
+In this example, we'll assume that this is the first time we've tried to install
+a printer, which will show the following:
+
+|add printer|
+
+Click "Add a Printer". By doing so, you'll now get a list of printers that Tails
+has auto-detected. You should now see this:
+
+|select printer to add|
+
+In this example, we've connected an HP Deskjet F4200. Clicking on this printer
+will select it for installation, which, if successful, will display the
+following:
+
+|printer installing|
+
+This indicates the Tails is attempting to install the USB printer; assuming you
+receive no errors in this process, you will then see the following screen,
+which indicates that the printer is "ready" for printing.
+
+|printer ready|
+
+For instances where your printer does *not* work out of the box, most difficulties
+stem from not selecting the right driver (extra software needed for the printer
+and computer to communicate). Luckily, Tails has a large number of drivers for
+just about any popularly manufactured printer on hand, without even having to
+download new drivers from the web.
```

Comment:
This seems like a natural break point where it would be appropriate to add another section heading. The last section was a walk-through of adding a printer successfully, while the next section is about troubleshooting a printer that doesn't work out of the gate. I would add a "Troubleshooting" section heading here.